docs/sdk/src/guides/parachains/async_backing_guide.rs

```diff
@@ -0,0 +1,452 @@
+//! # Upgrade Parachain for Asynchronous Backing Compatibility
```

Comment:
What is the plan here will we make https://wiki.polkadot.network/docs/maintain-guides-async-backing point to here ? Because otherwise we will have to maintain this information in two places.

I don't think at this point we can remove https://wiki.polkadot.network/docs/maintain-guides-async-backing, because it has references in other articles and forum posts and it would be a bad experience if it becomes a broken link.

Reply:
Hey @alexggh That is the plan. We will retain preliminary content on that page and add a prominent link to the Polkadot SDK doc. The Wiki doc link will not be broken.